test/rules/no-fallthrough.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+	valid: [
+		"switch(foo) {\n" +
+		"    case 1:\n" +
+		"        doSomething();\n" +
```

Comment:
This should not be valid because there are no curlies around each case. I understand that you're not detecting these yet, but that means it should be a commented out error case, not a success case.

Reply:
Ah! I don't think I explained this, but I was intending for each of these tests to be independent, so that what is valid or invalid depends only on the rule that it is testing. In this case, the curlies would be a separate rule since I was testing for fallthrough. Let me know if you think I should change this approach!

Reply:
That seems fine.

Reply:
Ok, great!